scripts/s3BucketOptions/setBucketLifecyclePolicy.py

```diff
@@ -41,6 +41,7 @@ def main(
         print("########")
     except botocore.exceptions.ClientError as e:
         if e.response["Error"]["Code"] == "NoSuchLifecycleConfiguration":
+            print("Lifecycle Settings before the change:")
```

Comment:
Use loggers instead of prints to display info. You will have a chance in the future to capture structured log messages in the future

Reply:
good point thanks for the input. feel free to open a ticket about it :) code cleanup is imho out of the scope of this minor PR :)

Reply:
good point thanks for the input. feel free to open a ticket about it :) code cleanup is imho out of the scope of this minor PR :)

Reply:
https://github.com/ITISFoundation/osparc-ops-environments/issues/97